internal/services/bot/bot_channel_web_chat_resource.go

```diff
@@ -248,6 +377,42 @@ func flattenSiteNames(input *[]botservice.WebChatSite) []interface{} {
 	return results
 }
 
+func flattenSites(input *[]botservice.WebChatSite) []interface{} {
+	results := make([]interface{}, len(*input))
```

Comment:
We should explicitly initialise it with 0 length
```suggestion
	results := make([]interface{}, 0)
```

Reply:
updated